arrows/stanag/segments/stanag_4607_segment_lookup.cxx

```diff
@@ -0,0 +1,205 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "stanag_4607_segment_lookup.h"
+
+namespace ka = kwiver::arrows;
+
+#include <vital/exceptions/base.h>
+
+#include <ostream>
+#include <initializer_list>
+#include <vector>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace stanag {
+
+// ----------------------------------------------------------------------------
+stanag_4607_segment_type_traits_lookup const&
+stanag_4607_segment_type_traits_lookup_table()
+{
+  static stanag_4607_segment_type_traits_lookup const lookup = {
+    {
+      STANAG_4607_SEGMENT_TYPE_MISSION,
+      "STANAG_4607_SEGMENT_TYPE_MISSION",
+      std::make_shared< stanag_4607_mission_segment_format >(),
+      "Mission Segment"
+    },
+    {
+      STANAG_4607_SEGMENT_TYPE_DWELL,
+      "STANAG_4607_SEGMENT_TYPE_DWELL",
+      std::make_shared< stanag_4607_dwell_segment_format >(),
+      "Dwell Segment"
+    }
+    // TODO finish rest of segments
```

Comment:
I would recommend updating the comment that we currently are supporting a subset of the standard with plans on adding more support in time.